### ISSUE TITLE
Audio popping sound on boot/sleep/shutdown AppleALC 1.6.0 layout 18

### ISSUE THREAD

**Issue by lambdaupb** [viorel78/ASRock-DeskMini-310#1]
I have loud popping sounds on my speaker connected to the front audio out when booting, on sleep or shutdown with the DeskMini 310 on 10.15.7.
Before I used layout 28, which worked also, but had the same popping sounds.

I saw that you contributed the layout 18 to AppleALC and tried it, but same result on OpenCore 0.6.9 and AppleALC 1.6.0.

I use the https://github.com/appleserial/DeskMini as a base, but I spliced the ACPI from your repo to see if that is the reason, but no difference.

Do you have the same issue?

**Comment by viorel78** [comment]
Yes, I know,  the resources for Layout 18 I did, after 3 months I have been trying to fix it, but so far with no results, there is an option to turn off the second microphone to get rid of the pop-up, but more tests need to be done, if it will works, I will add it to the Apple Alc kext

**Comment by lambdaupb** [comment]
> ... there is an option to turn off the second microphone to get rid of the pop-up ...

How do I disable the mic? I don't need it and would prefer no loud noise on boot/shutdown instead.
Is it enough to delete the row with the mic in PinConfigurator?

Thanks for taking the time!

**Comment by viorel78** [comment]
try this kext https://drive.google.com/file/d/1Dz3s_wsA_oFt4fiQy60LWnC58uifXs4v/view?usp=sharing

**Comment by lambdaupb** [comment]
Thanks for taking the time.

I tried the kext, but the popping on boot/shutdown is still there.

------
I edited layout18 with PinConfigurator and deleted all but the Front Line-Out, that makes the popping a little bit less, but it is still there.

**Comment by viorel78** [comment]
bought and connected the rear audio cable - no popping on boot/shutdown

**Comment by lambdaupb** [comment]
Oh, that may be a good alternative, thanks for the info!
